Makefile.vars.mk

```diff
@@ -50,4 +50,4 @@ KUBENT_IMAGE    ?= ghcr.io/doitintl/kube-no-trouble:latest
 KUBENT_DOCKER   ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(root_volume) --entrypoint=/app/kubent $(KUBENT_IMAGE)
 
 instance ?= defaults
-test_instances = tests/defaults.yml tests/full.yml
+test_instances = tests/defaults.yml tests/legacy.yml
```

Comment:
Generally, you should add additional tests with `commodore component update -t <testName> .`. That way, you'll ensure that all the component boilerplate (GitHub actions, Makefile, etc.) are updated to know about the new test case.